{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--79510ec2-eedd-46d2-a9df-3673bb7a46d8",
    "created": "2024-08-13T16:04:08.611857Z",
    "modified": "2024-08-13T16:04:08.611862Z",
    "name": "No Title Available",
    "description": "libautotrace.a in AutoTrace 0.31.1 allows remote attackers to cause a denial of service (invalid read and SEGV), related to the ReadImage function in input-bmp.c:425:14.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2017-9179"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://blogs.gentoo.org/ago/2017/05/20/autotrace-multiple-vulnerabilities-the-autotrace-nightmare/"
        }
    ]
}